packages/foreman/libsass/libsass.spec

```diff
@@ -0,0 +1,137 @@
+Name:           libsass
+Version:        3.6.4
+Release:        5%{?dist}
+Summary:        C/C++ port of the Sass CSS precompiler
+
+License:        MIT
+URL:            http://sass-lang.com/libsass
+Source0:        https://github.com/sass/libsass/archive/%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires:  automake
+BuildRequires:  autoconf
+BuildRequires:  libtool
+BuildRequires:  pkgconfig
+BuildRequires:  gcc-c++
+BuildRequires: make
+
+%description
+Libsass is a C/C++ port of the Sass CSS precompiler. The original version was
+written in Ruby, but this version is meant for efficiency and portability.
+
+This library strives to be light, simple, and easy to build and integrate with
+a variety of platforms and languages.
+
+Libsass is just a library, but if you want to RUN libsass, install the sassc
+package.
+
+
+%package        devel
+Summary:        Development files for %{name}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description    devel
+The %{name}-devel package contains libraries and header files for
+developing applications that use %{name}.
+
+
+%prep
+%setup -q
+export LIBSASS_VERSION=%{version}
+autoreconf --force --install
+
+
+%build
+%configure --disable-static
+make %{?_smp_mflags}
+
+
+%install
+make install DESTDIR=$RPM_BUILD_ROOT
+find $RPM_BUILD_ROOT -name '*.la' -exec rm -f {} ';'
+
+
+%ldconfig_scriptlets
```

Comment:
Is this supported in EL8? https://docs.fedoraproject.org/en-US/epel/epel-packaging/#shared_libraries phrasing implies it is, but it's not explicit.

Reply:
Kinda sure, yeah.

But will double check

Reply:
Interestingly, on both EL9 and EL8 this ends up generating nothing:

```
[root@15b1f7780590 /]# rpm -q --scripts libsass
[root@15b1f7780590 /]# 
```